tools/pipelines/build-client.yml

```diff
@@ -97,3 +98,16 @@ extends:
           node node_modules/@fluidframework/build-tools/dist/genMonoRepoPackageJson/genMonoRepoPackageJson.js
           cp repo-package.json packages/package.json
           cp repo-package-lock.json packages/package-lock.json
+
+    additionalTests:
+      - task: Npm@1
+        displayName: '[end-to-end tests] npm run ci:test:realsvc'
+        env:
+          fluid__webpack__bearerSecret: $(fluid__webpack__bearerSecret)
+          fluid__webpack__fluidHost: $(fluid__webpack__fluidHost)
+          fluid__webpack__tenantId: $(fluid__webpack__tenantId)
+          fluid__webpack__tenantSecret: $(fluid__webpack__tenantSecret)
+        inputs:
+          command: 'custom'
+          workingDir: packages/test/end-to-end-tests
+          customCommand: 'run ci:test:realsvc' # This script ends with 'exit 0' so this task should never fail
```

Comment:
This is not running with nyc.  So it won't have code coverage. data generated

Reply:
The normal test get nyc from the root package.json invoking with `nyc npm run test:report`

---
In reply to: [490429658](https://github.com/microsoft/FluidFramework/pull/3612#discussion_r490429658) [](ancestors = 490429658)

Reply:
Yeah I had test coverage written down as a follow-up. Ok to leave out of this PR?